packages/ui/src/components/Channel/index.tsx

```diff
@@ -243,6 +243,18 @@ export function Channel({
     ]
   );
 
+  const handleGoBack = useCallback(() => {
+    if (
+      draftInputPresentationMode === 'fullscreen' &&
+      draftInputRef.current != null
+    ) {
+      draftInputRef.current.exitFullscreen();
+      setEditingPost?.(undefined);
+    } else {
```

Comment:
what about the case of not fullscreen but `draftInputRef.current != null`, would we need to also `setEditingPost?.(undefined)`?